packages/angular/build/tools/baseline_browserslist/tsconfig-build.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Rename the file to `tsconfig.json` so that the IDE can also pick it up as otherwise the config in the root tsconfig are applied when opening the nested files.